speechtotextv1/speech_to_text_v1_adapter.go

```diff
@@ -134,12 +134,6 @@ func (speechToText *SpeechToTextV1) RecognizeUsingWebsocket(recognizeWSOptions *
 	if recognizeWSOptions.BaseModelVersion != nil {
 		param.Set("base_model_version", *recognizeWSOptions.BaseModelVersion)
 	}
-	if recognizeWSOptions.ProcessingMetrics != nil {
-		param.Set("processing_metrics", fmt.Sprint(*recognizeWSOptions.ProcessingMetrics))
-	}
-	if recognizeWSOptions.ProcessingMetricsInterval != nil {
-		param.Set("processing_metrics_interval", fmt.Sprint(*recognizeWSOptions.ProcessingMetricsInterval))
-	}
```

Comment:
These can stay in `RecognizeUsingWebsocket`. They cannot be in the POST recognize operation

Reply:
This is only removing from the url as it is not a url param